include/nbl/asset/utils/IGLSLCompiler.h

```diff
@@ -27,9 +27,10 @@ class IGLSLCompiler final : public core::IReferenceCounted
 {
 		core::smart_refctd_ptr<IIncludeHandler> m_inclHandler;
 		system::ISystem* m_system;
+		system::logger_opt_smart_ptr m_logger;
```

Comment:
dont hold a logger, make it possible to have a different logger on each compilation